_data/master/navbars/reference.yml

```diff
@@ -113,8 +113,19 @@ toc:
     path: /reference/architecture/data-path
 - title: Advanced
   section:
-  - title: RBAC for etcdv2
-    path: /reference/advanced/etcd-rbac
+  - title: Guide for etcd RBAC
```

Comment:
Yeah agreed it isn't inline with others.

Adding etcdv2 wouldn't be appropriate because it actually covers v3 also.